### DISCUSSION TITLE
Accrescent Store?

### DISCUSSION THREAD

**Discussion by ghost** [WrichikBasu/ShakeAlarmClock#86] [General]
Hey,

could you publish Shake Alarm Clock in the Accrescent Store? I would love to see you there and download the App from there.

Maybe you ask why we want this or what Benefits it would have for us as Users.

The Benefits for me would be a few, like these:

I could get Shake Alarm Clock from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

It would make it a lot easier for me and save me time, because Mastodon is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

I really hope that you Guys could make that possible, because it would have great Benefits for us as Users. And think like that: If we support them by adding Apps, we help them to grow from Alpha to Stable.

[Accresent features](https://accrescent.app/features) I think matter for Shake Alarm Clock.
 
Security: 
[Signed repository metadata](https://accrescent.app/features#signed-metadata) 
[App signing key pinning](https://accrescent.app/features#key-pinning) 
[Name verification](https://accrescent.app/features#name-icon-verification)
 
Privacy: 
[No remote APK signing](https://accrescent.app/features#no-remote-signing) 
[No accounts](https://accrescent.app/features#no-accounts)

**Comment by WrichikBasu** [comment]
I'll have a look; until then, you can always use Izzy on Droid. Each and every app on Izzy's repo is thoroughly checked, so apps downloaded from there are pretty secure too. The link for this app is given in the `README.md` file.

**Reply by ghost** [reply]
Okay, thank you so much! Yes, that's true. Okay, thank you.

**Comment by WrichikBasu** [comment]
I have decided not to host it on any other store at the moment. If you believe that the app has malware, I would recommend not using it.
